.github/dependabot.yml

```diff
@@ -12,6 +12,6 @@ updates:
       interval: "daily"
   # Requirements to run style checks
   - package-ecosystem: "pip"
-    directory: "/.github/workflows/style"
+    directory: "/.github/workflows/requirements/*"
```

Comment:
From https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directories

> The directories key supports globbing and the wildcard character *. These features are not supported by the directory key.

So, I guess either we spell out the directory without `*` or we use `directories`

Reply:
Ah good catch. I switched to `directories` so we can make this more general in the future and put other requirements in there more easily.